compiler/one-cmds/tests/prepare_test_materials.sh

```diff
@@ -119,7 +126,7 @@ function files_missing() {
 
 declare -a TEST_RECCURENT_MODELS=(\
   "RNN.onnx" "RNN-nobias.onnx" "RNN-relu.onnx" "RNN-bi.onnx" "RNN-noinit.onnx"\
-  "LSTM.onnx" "LSTM-bi.onnx" "LSTM-noinit.onnx" "LSTM-nobias.onnx"
+  "LSTM.onnx" "LSTM-bi.onnx" "LSTM-noinit.onnx" "LSTM-nobias.onnx" "Conv2D_000.circle"
```

Comment:
Conv2D_000.circle is reccurent model?

Reply:
😓 I misread the script. I'll remove it. Thank you for pointing out!